resources/packs/terraform/tfplan.go

```diff
@@ -84,7 +84,7 @@ func (t *mqlTerraformPlan) GetResourceChanges() ([]interface{}, error) {
 			}
 		}
 
-		var replacePaths map[string]interface{}
+		var replacePaths []interface{}
```

Comment:
Great. Can we add a test for that specific case?

Reply:
Sure. I added one to check this resource.